tests/conftest.py

```diff
@@ -263,8 +266,24 @@ def factory(
             pvc_obj = PVC(**custom_data)
             pvc_obj.create(do_reload=False)
         else:
-            project = project or project_factory()
-            storageclass = storageclass or storageclass_factory(interface)
+            nonlocal active_project
+            nonlocal active_rbd_storageclass
+            nonlocal active_cephfs_storageclass
+
+            project = project or active_project or project_factory()
```

Comment:
Cannot we squash this and line below into one line? active_project = project or active_project or project_factory()
And then use just active project later on? I think only one reference on line to 291 project.namespace.

Reply:
Right, good point